reference/configuration/framework.rst

```diff
@@ -1298,6 +1298,19 @@ activate the data collectors manually::
 
     $profiler->enable();
 
+collect_parameter
+.................
+
+**type**: ``string`` **default**: ``null``
+
+This option allows to enable or disable the profiler on a per request basis
```

Comment:
Are you sure this is the description for the collect**_parameter** option and not for the `collect` option above?

Reply:
I've changed the first sentence, hopefully it's less confusing right now.